clustermesh-apiserver/health.go

```diff
@@ -58,8 +58,13 @@ func registerHealthAPIServer(lc hive.Lifecycle, clientset k8sClient.Clientset, c
 		OnStart: func(hive.HookContext) error {
 			go func() {
 				log.Info("Started health API")
-				if err := srv.ListenAndServe(); err != nil {
-					log.WithError(err).Fatalf("Unable to start health API")
+				err := srv.ListenAndServe()
+				switch err {
+				case http.ErrServerClosed:
+					log.Info("Health API shutdown successfully")
```

Comment:
s/fully/full/. Though do we need to log this? Hive logs the call to the OnStop already.

Reply:
I've removed the log message as you suggested.